tests/aws-cpp-sdk-kinesis-integration-tests/KinesisTests.cpp

```diff
@@ -171,4 +172,106 @@ TEST_F(KinesisTest, EnhancedFanOut)
     AWS_ASSERT_SUCCESS(m_client->DeregisterStreamConsumer(deregisterRequest));
 }
 
+
+void WriteDataToStream(Aws::Kinesis::KinesisClient &kinesis_client, const std::string &streamName, const std::string &data, const std::string &partitionKey)
+{
+    std::cout<<"WriteDataToStream called"<<std::endl;
```

Comment:
please remove debug traces, our tests typically do not trace to stdout on success.
you can use SCOPED_TRACE() macro of gtest to add additional context to the test that will be dumped to stdout on failure.